### PR TITLE
Add new package for pgroll migrations

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: packages/pgroll/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "generate": "tsx scripts/build.ts",
```
Comment (SferaDev) [inline packages/pgroll/package.json:17]: We will need to make sure that GitHub Actions keeps it up to date

File: packages/pgroll/.npmignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+src
+tsconfig.json
+rollup.config.mjs
+.eslintrc.cjs
+.gitignore
```
Comment (SferaDev) [inline packages/pgroll/.npmignore:5]: We should npmignore pgroll.schema.json

**Review by eemmiillyy — DISMISSED** [review]

File: packages/pgroll/src/json-schema.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const PGROLL_JSON_SCHEMA_URL = 'https://raw.githubusercontent.com/xataio/pgroll/fix-schema/schema.json';
```
Comment (SferaDev) [inline packages/pgroll/src/json-schema.ts:1]:
```suggestion
export const PGROLL_JSON_SCHEMA_URL = 'https://raw.githubusercontent.com/xataio/pgroll/main/schema.json';
```

File: packages/pgroll/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { schema } from './schema';
+
+export * from './types';
+
+export const PGROLL_JSON_SCHEMA_URL = 'https://raw.githubusercontent.com/xataio/pgroll/fix-schema/schema.json';
```
Comment (SferaDev) [inline packages/pgroll/src/index.ts:5]:
```suggestion
export const PGROLL_JSON_SCHEMA_URL = 'https://raw.githubusercontent.com/xataio/pgroll/main/schema.json';
```

**Review by eemmiillyy — APPROVED** [review]

**Pull request merged**